scripts/helmcharts/openreplay/charts/frontend/templates/deployment.yaml

```diff
@@ -43,6 +43,8 @@ spec:
           {{- .Values.healthCheck | toYaml | nindent 10}}
           {{- end}}
           env:
+            - name: RECYCLE
```

Comment:
This "RECYCLE" is just an inert env var we can bump to cause people to pull the new docker image when the pods are recycled due to this variable being modified.